packages/vuetify/src/components/VExpansionPanel/VExpansionPanel.sass

```diff
@@ -1,99 +1,86 @@
+        opacity: $expansion-panel-disabled-overlay / $expansion-panel-disabled-opacity
 
 // Block
 .v-expansion-panels
-  border-radius: $expansion-panel-border-radius
   display: flex
   flex-wrap: wrap
   justify-content: center
   list-style-type: none
   padding: 0
   width: 100%
-  z-index: 1
 
-  > *
-    cursor: auto
+  &:not(&--variant-accordion)
+    > :not(:first-child):not(:last-child):not(.v-expansion-panel--active):not(.v-expansion-panel--before-active)
+      border-bottom-left-radius: 0 !important
+      border-bottom-right-radius: 0 !important
 
-  > *:first-child
-    border-top-left-radius: inherit
-    border-top-right-radius: inherit
+    > :not(:first-child):not(:last-child):not(.v-expansion-panel--active):not(.v-expansion-panel--after-active)
+      border-top-left-radius: 0 !important
+      border-top-right-radius: 0 !important
 
-  > *:last-child
-    border-bottom-left-radius: inherit
-    border-bottom-right-radius: inherit
+    > :first-child:not(.v-expansion-panel--active):not(.v-expansion-panel--before-active)
+      border-bottom-left-radius: 0 !important
+      border-bottom-right-radius: 0 !important
 
-  // round corners on non-accordion/tile panels
-  &:not(&--accordion):not(&--tile)
-    > .v-expansion-panel
-      &--active
-        border-radius: $expansion-panel-border-radius
+    > :last-child:not(.v-expansion-panel--active):not(.v-expansion-panel--after-active)
+      border-top-left-radius: 0 !important
+      border-top-right-radius: 0 !important
 
-        + .v-expansion-panel
-          border-top-left-radius: $expansion-panel-border-radius
-          border-top-right-radius: $expansion-panel-border-radius
+  &--variant-accordion
+    > :first-child
+      border-bottom-left-radius: 0 !important
+      border-bottom-right-radius: 0 !important
 
-      &--next-active
-        border-bottom-left-radius: $expansion-panel-border-radius
-        border-bottom-right-radius: $expansion-panel-border-radius
+    > :last-child
+      border-top-left-radius: 0 !important
+      border-top-right-radius: 0 !important
 
-        .v-expansion-panel-header
-          border-bottom-left-radius: inherit
-          border-bottom-right-radius: inherit
+      .v-expansion-panel-header--active
+        border-bottom-left-radius: initial
+        border-bottom-right-radius: initial
+
+    > :not(:first-child):not(:last-child)
+      border-radius: 0 !important
+
+    .v-expansion-panel-header__overlay
+      transition: 0.3s border-radius $standard-easing
 
 // Element
 .v-expansion-panel
   flex: 1 0 100%
   max-width: 100%
   position: relative
-  transition: .3s map-get($transition, 'swing')
-
-  &::before
-    border-radius: inherit
-    bottom: 0
-    content: ''
-    left: 0
-    position: absolute
-    right: 0
-    top: 0
-    z-index: -1
-    +elevationTransition()
-    +elevation(2)
+  transition: .3s all $standard-easing
+  transition-property: margin-top, border-radius, border, max-width
```

Comment:
move to sass variables

Reply:
not sure I agree. this is very specific to how the expansion panel works. setting different transition properties will break stuff.